internal/testplanet/planet.go

```diff
@@ -267,7 +279,19 @@ func (planet *Planet) newSatellites(count int) ([]*satellite.Peer, error) {
 				MaxBufferMem:  4 * memory.MB,
 				APIKey:        "",
 			},
-			// TODO: Audit    audit.Config
+			Audit: audit.Config{
+				MaxRetriesStatDB: 0,
+				Interval:         30 * time.Second,
+			},
+			Tally: tally.Config{
+				Interval: 30 * time.Second,
```

Comment:
can this be reconfigured somewhere?

Reply:
This is the testplanet config, but at the moment no. For the satellite binary it takes the argument from flags.

For testplanet I have two ideas with regards to disabling/adjusting configuration:
1. hooks for config (e.g. you could specify callbacks to change the config before creating the peer)
2. flags for disabling specific subsystems (e.g. you want to manually play with manually and you don't want to run it)